tests/inc/az_ulib_test_my_interface.h

```diff
@@ -91,46 +93,12 @@ extern "C"
     const az_ulib_interface_descriptor* descriptor;
     uint32_t wait_policy_ms;
     az_ulib_ipc_interface_handle handle;
-    az_ulib_capability_index command_index;
+    az_ulib_capability_index capability_index;
     az_result return_result;
   } my_command_model_in;
 
   typedef az_result my_command_model_out;
 
-/*
- * Define model of my_command_async in my_interface.
```

Comment:
I remember you saying that you were going to hold off on async capability until later. I see it was included earlier, is it still possible to use async commands with the generic capability interface?

Reply:
I'll put it back whene we decide to implent it for real.